src/app/FakeLib/FakeLib.fsproj

```diff
@@ -306,6 +306,9 @@
     <Compile Include="UnitTest\MSTest.fs" />
     <Compile Include="UnitTest\ProcessTestRunner.fs" />
     <Compile Include="UnitTest\VSTest.fs" />
+    <Compile Include="..\Fake.DotNet.Testing.MSTest\MSTest.fs">
+      <Link>Fake.DotNet.Testing.MSTest/MSTest.fs</Link>
+    </Compile>
```

Comment:
Can we move that up to the other "Modules"? This prevents accidental usage of "old" members (and makes the IDE help you)